### PR TITLE
setup firebase sdk, pom.xml and test code to write to db

### PR DESCRIPTION
**Pull request by leafgecko**

### CODE REVIEW

File: my-app/ServiceAccountKey.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (EugeneTeu) [inline my-app/ServiceAccountKey.json]: need to remove from version control as discussed

**Review by pohlinwei — COMMENTED** [review]
Is there a reason for pushing the .class files? 😅

**Review by EugeneTeu — CHANGES_REQUESTED** [review]
remove the api key plox

**Comment by leafgecko** [comment]
> Is there a reason for pushing the .class files?

Oh that was auto generated as part of maven i believe, should we delete those or leave it there?

**Review by EugeneTeu — APPROVED** [review]

**Review by pohlinwei — APPROVED** [review]

**Pull request merged**